packages/docs/content/1_getting-started/0_installation.mdx

```diff
@@ -72,6 +74,64 @@ Copy the files and folders from `download/design-system/dist` to a relevant plac
     └── index.html
 ```
 
+## Utilize the CDN
+
+A content delivery network (CDN) is a collection of servers that dynamaically allocate resources dependent on the requestors geographic location in order to optimize the distribution of assets.
+
+Some benefits and applications for utilizing the design system via CDN are:
+
+- Quickly utilize design system code and style assets for development/prototyping
+- The CDN caches assets for fast loading and scales automatically in case of a large number of requests
+- Ability to utilize a specific version of the design system easily
+
+### URL structure
+
+```
+https//design.cms.gov/cdn/<package-name>/<version>/<resource>
+```
+
+### Packages
+
+Packages which are currently availble for reference:
+
+- **design-system** - The core CMSDS
+- **ds-healthcare-gov** - Healthcare.gov themed DS
+- **ds-medicare-gov** - Medicare.gov themed DS
+
+### Available versions
+
+Versions numbers `3.4.0` and higher are currently available via CDN.
```

Comment:
It might also be worth mentioning the minimum version numbers for the two child design systems

Reply:
Good call, moved to the packages section above.